docs/get_started.md

```diff
@@ -110,32 +110,51 @@ the creation of a new management pack project.
     Along with additional language-specific configuration.
     Currently, `mp-init` offers the following languages:
 
-    `Python`
+    - `Python`
  
-    `Java` 
+    - `Java` 
+ 
+9.  === "Python"
+ 
+        a. `Select a template for your project`
 
-9. `(For Java projects only) Enter package name: `
+           Both of the available options will generate a project structure that can be modified into an adapter:
+
+           `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
+           The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+
+           `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
+           connection, collection, adapter definition, and endpoints logic.
 
-    The package name will be used to set up the package used by the adapter and the directory structure of the project.
-   
-    ???+ example 
- 
-        ``` java
-        package com.mycompany;
-        ```
 
-10. `Select a template for your project`
+    === "Java"
 
-    Both of the available options will generate a project structure that can be modified into an adapter:
+        a. `Enter package name: `
 
-    `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
-    The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+           The package name will be used to set up the package used by the adapter and the directory structure of the project.
+           
+        ???+ example 
+     
+            ``` java
+            package com.mycompany;
+            ```
 
-    `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
-    connection, collection, adapter definition, and endpoints logic.
+        b. `Select a template for your project`
+
+           Both of the available options will generate a project structure that can be modified into an adapter:
+
+           `Sample Adapter`: Creates a template adapter that collects several objects and metrics from the container the adapter is running.
+           The template adapter has comments throughout its code that explain what the code does and how to customize it for your adapter.
+
+           `New Adapter`: Creates methods, minimal comments, and code necessary to implement test
+           connection, collection, adapter definition, and endpoints logic.
+
+!!! warning "" 
 
     For the purposes of this *Get Started* guide, select **Sample Adapter**.
 
+---
+
```

Comment:
The way this is formatted doesn't read as part of the step it is for. I think it would be better to duplicate and have it as part of each language's respective step. I'm also not sure that putting it in a warning box is appropriate, even without the 'warning' text/icon

Reply:
This is much better 👍